### DISCUSSION TITLE
About sorting items

### DISCUSSION THREAD

**Discussion by DuckyZebra** [moudey/Shell#513] [General]
Hi!
Windows always had that bad habit to give us sorting options which does not fit our needs and habits and it is usually Sort with Data, (which is not very good for giving proper results)
**I wish we would have always these four types:**
Sort by Name, Type, **Modification Date** and Size. Is this possible?

Thank you for your great menu tool.

**Comment by RubicBG** [comment]
I personally don't understand where you want the sorting to be? what do you want to do with this sort? Explain more, give an example

**Reply by DuckyZebra** [reply]
![obraz](https://github.com/user-attachments/assets/9529875e-a1e6-43cd-8d35-2bd0c3e82bc3)
Hi! I would like to always have the sorting option by Modification Date rather than Date. Modification Date always gave me best results.

**Reply by RubicBG** [reply]
NS can only control / manage the "windows" context menu, not the explorer view. 

ps: you can take a look at this https://lesferch.github.io/WinSetView/ (never used)

**Reply by Eta-Beta** [reply]
Hello, if it can help I have been using WinSetView for a long time and I am very satisfied with it. Windows of all versions, has always done what it wanted with the views, but worse, it has never done anything to solve this problem. In the Microsoft community they have always limited themselves to saying that it was enough to set the preference once and then it was remembered, but then it did what it wanted. With WinSetView you can set each main folder to display it as you want.
